composer/datasets/webdataset_utils.py

```diff
@@ -15,10 +15,10 @@
 from tqdm import tqdm
 
 if TYPE_CHECKING:
-    from webdataset import WebDataset
+    from webdataset import WebDataset  # type: ignore
```

Comment:
Why do we have these type ignores?

Reply:
Good Q; see https://github.com/mosaicml/composer/pull/1001/files#r866082405 

#1050 will be removing these type ignores